scripts/x-data-grid-premium.exports.json

```diff
@@ -128,9 +128,11 @@
   { "name": "GridCellModes", "kind": "Enum" },
   { "name": "GridCellModesModel", "kind": "TypeAlias" },
   { "name": "GridCellParams", "kind": "Interface" },
-  { "name": "GridCellProps", "kind": "Interface" },
+  { "name": "GridCellProps", "kind": "TypeAlias" },
   { "name": "GridCellSelectionApi", "kind": "Interface" },
   { "name": "GridCellSelectionModel", "kind": "TypeAlias" },
+  { "name": "GridCellV7", "kind": "Variable" },
+  { "name": "GridCellWrapper", "kind": "Variable" },
```

Comment:
Is there a need to export these components?

Reply:
There are often `export * ...` statements, I could adjust them to restrict what is exported but it also messes with the imports in other places.

E.g. the following one would also need to be adjusted. I feel it would be less of a burden to maintain to keep things as they are, but I can update if needed.

https://github.com/mui/mui-x/blob/1d611c02e624ee63d358798914c0e890c81190d1/packages/grid/x-data-grid/src/constants/defaultGridSlotsComponents.ts#L2-L15

Reply:
Yeah, we need to change exports in multiple places.
I generally prefer to avoid additional exports if possible, so that we can make internal changes without worrying about breaking changes 🙂
So if there's no need to export these components - let's not do it